codalab/worker/main.py

```diff
@@ -138,7 +138,7 @@ def parse_args():
         '--checkin-frequency-seconds',
         help='Number of seconds to wait between worker check-ins',
         type=int,
-        default=5,
+        default=20,
```

Comment:
I still think it would be nice to update the worker status at least once every 5 seconds...we don't need this to be 20, right?

Reply:
No -- we can keep it at 5